commands/account/createOverride.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import { getCwd } from '@hubspot/local-dev-lib/path';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { getConfigPath, getAccountId } from '@hubspot/local-dev-lib/config';
+import { addConfigOptions } from '../../lib/commonOpts';
+import { i18n } from '../../lib/lang';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { selectAccountFromConfig } from '../../lib/prompts/accountsPrompt';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
+
+const i18nKey = 'commands.account.subcommands.createOverride';
+
+export const describe = null; // i18n(`${i18nKey}.describe`);
+
+export const command = 'create-override [account]';
+
+type AccountInfoArgs = CommonArgs &
+  ConfigArgs & {
+    account?: string | number;
```

Comment:
Account can always be a string here, which you can declare when you define the positional